sdk/messaging/azservicebus/liveTestHelpers_test.go

```diff
@@ -27,15 +27,12 @@ func setupLiveTest(t *testing.T, props *admin.QueueProperties) (*Client, func(),
 	testCleanup := func() {
 		require.NoError(t, serviceBusClient.Close(context.Background()))
 		cleanupQueue()
-		require.NoError(t, serviceBusClient.Close(context.Background()))
```

Comment:
Does closing twice return an error now?

Reply:
It just seemed unnecessary but it shouldn't cause an error (well, it never has, even now). But maybe I should keep it, why not. I'll add it back.

Reply:
Added it back and popped a comment on there.